runtimes/common/src/constants.rs

```diff
@@ -430,6 +430,21 @@ pub mod fee {
 	}
 }
 
+pub mod public_credentials {
+	use super::*;
+
+	/// The size is checked in the runtime by a test.
+	pub const MAX_PUBLIC_CREDENTIAL_STORAGE_LENGTH: u32 = 257;
```

Comment:
Might be a dump question but I suppose this shall represent that the length should not exceed 2^8 = 256? Since we allow for equality, shouldn't it be 256 then?

Reply:
No, this is the size of an encoded storage entry for a public credential. This has been generated by the runtime tests. It has nothing to do with parameters constraints, but it is used to calculate the storage deposit. Or am I missing your point?